lib/diplomat/rest_client.rb

```diff
@@ -17,6 +17,19 @@ def use_named_parameter(name, value)
       value ? ["#{name}=#{value}"] : []
     end
 
+    def use_consistency(options, default_value = [])
```

Comment:
I don't get the point of having a default value if it's always set to nil, and checked if its value is not nil... Better just remove the `if use_consistency(options, nil)` and use the default value

Reply:
It was to avoid modifying lib/diplomat/kv.rb, but I did the other way around finally in next patch